decentraland/unity-explorer
# chore: Adjusted quality settings, pp effects and render features

## What does this PR change?

- Postprocessing volume of the scene has been removed
- Postprocessing effects are now tied to quality settings
- Moved bloom and color adjustment to quality pp
- Quality settings now have different values for low medium and high
- Render features are now adjusted and disables / enabled based on quality settings

## How to test the changes?

1. Open the debug menu and check your performance
2. Open the settings menu and change between Quality settings
3. Compare performance between them
4. Check any visual glitch that wasnt ther before
